### PR TITLE
Exemptions changes

### PR DESCRIPTION
**Pull request by Bnaya**

### CODE REVIEW

**Comment by Bnaya** [comment]
@akariv We've managed to add the exemption code with the merge problems + more features
CC @naamac @shayshei

File: open-budget/app/scripts/entity_details.coffee [hunk 1]
```diff
@@ -1,14 +1,17 @@
-define(["jquery", "backbone", "models", "templates"], ($, Backbone, models, JST) ->
+define(["jquery", "underscore", "backbone", "models", "templates"], ($, _, Backbone, models, JST) ->
     class EntityDetailsView extends Backbone.View
             events:
               'click .exemption-expander': 'toggleExemptionDetails'
 
             initialize: ->
-                @model.selectedEntity.on 'change:selected', =>
```
Comment (akariv) [inline open-budget/app/scripts/entity_details.coffee]: @Bnaya @naamac @shayshei 

Did you make sure that the stand-alone entity page is not affected by this change?

Comment (akariv) [inline open-budget/app/scripts/entity_details.coffee]: Okay, just tested it - it seems the stand-alone entity pages are indeed broken...

Comment (Bnaya) [inline open-budget/app/scripts/entity_details.coffee]: what is the url for stand-alone entity pages?

Comment (akariv) [inline open-budget/app/scripts/entity_details.coffee]: Here's an example:
http://www.obudget.org/g2/en/511380693#entity/511380693/2014/main

I put the comment on the change that probably broke it, but I didn't test it to make sure.

**Comment by Bnaya** [comment]
@akariv The last commit fix that issue

**Pull request merged**